tiledb/core.cc

```diff
@@ -801,9 +801,6 @@ class PyQuery {
     tiledb_datatype_t type;
     uint32_t cell_val_num;
     std::tie(type, cell_val_num) = buffer_type(name);
-    uint64_t cell_nbytes = tiledb_datatype_size(type);
```

Comment:
Unused, revealed by a compiler warning -- not related to this PR

Reply:
Thanks